src/routes/tools/extended-drought/_ChartTitle.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+  import { Button } from "carbon-components-svelte";
+  import { Location16 } from "carbon-icons-svelte";
+
+  export let title;
+  export let subtitle;
+  export let climvarLabel;
+  export let scenarioLabel;
+  export let periodLabel;
+  export let loadLocation;
+</script>
+
+<style>
+  div {
+    max-width: 70ch;
+  }
+
+  p {
+    color: var(--gray-80);
+  }
+</style>
+
+<Button
+  class="btn-change-location"
+  size="small"
+  icon="{Location16}"
+  kind="ghost"
+  on:click="{loadLocation}"
+>
+  Change Location
+</Button>
+<div class="h3">
```

Comment:
I'm thinking that semantically we might want to use `<h2>` for the chart title, given that it is sort of the title for the dashboard. They could still be styled using the `h3` class though.